GoogleForCreators/web-stories-wp
# Use Moveable in Shapes library

## Summary
Uses moveable in shapes library so that the draggable shape looks exactly like the element that is being created on the canvas. That will be bases for snapping later, too.


## Relevant Technical Choices



## To-do

~- [] Fix snapping~ -- WILL BE DONE SEPARATELY
- [x] Ensure the shape being dragged is the same size as the shape added when clicking on the element.
- [x] Adjust tests if necessary



## User-facing changes



## Testing Instructions
I
1. Click on a shape
2. Verify it's added on the canvas as expected

II
1. Drag a shape onto the canvas
2. Verify that the dragged shape displays exactly the same as the shape that's being added
3. Verify the shape is added to the correct position when dropping


---



Fixes #5336
